src/execution.cc

```diff
@@ -732,11 +753,13 @@ bool Execution::DoBuild() {
 void Execution::DumpMetrics() {
   g_metrics->Report();
 
-  printf("\n");
+  logger_->Info("\n");
   int count = (int)state_->paths_.size();
```

Comment:
Seems strange to cast to int here, since we're going to get converted to double in a moment just below.